src/qt/forms/debugwindow.ui

```diff
@@ -1079,7 +1079,7 @@
                <item row="1" column="0">
                 <widget class="QLabel" name="peerConnectionTypeLabel">
                  <property name="toolTip">
-                   <string>The type of peer connection:&lt;ul&gt;&lt;li&gt;Inbound Full/Block Relay: initiated by peer&lt;/li&gt;&lt;li&gt;Outbound Full Relay: default&lt;/li&gt;&lt;li&gt;Outbound Block Relay: does not relay transactions or addresses&lt;/li&gt;&lt;li&gt;Outbound Manual: added using RPC %1 or %2/%3 configuration options&lt;/li&gt;&lt;li&gt;Outbound Feeler: short-lived, for testing addresses&lt;/li&gt;&lt;li&gt;Outbound Address Fetch: short-lived, for soliciting addresses&lt;/li&gt;&lt;/ul&gt;</string>
+                   <string>The type of peer connection: %1</string>
```

Comment:
f3153dc08fa16aa3bbca52f67833e5c9fbe8e095

Just move the whole tooltip? Why have a bit here?

Reply:
> Why have a bit here?

In Qt Designer it makes obvious a tooltip presence.

Reply:
> Just move the whole tooltip? Why have a bit here?

That's what I started with in #163, see https://github.com/bitcoin-core/gui/pull/163#issuecomment-757140245

There have been a few iterations on this tooltip 😄

Reply:
I see, still not fond of the approach. Being a dynamic tooltip I think it's fine to not be in the .ui file. If someone adds a tooltip in the designer then it will be obvious that it's being replaced at runtime. Reading this code makes you check what is in the .ui, and you still have to check this code if you change what's in the .ui. At the end anyone that wants to change this still has to check both places.

I think that a good rule of thumb is to leave all static/constant stuff for .ui.

Reply:
I don't have a strong opinion and saw other tooltips with html markup completely in the .cpp file, but I also don't have Qt or Qt Designer experience. I'm happy to keep it as-is or move it, either way.